content/en/tracing/trace_collection/open_standards/otel_collector_datadog_exporter.md

```diff
@@ -10,319 +10,338 @@ further_reading:
-      # ...
-    ```
+This ensures that [Kubernetes Attributes Processor][14] which is used in [our config map][15] is able to extract the necessary metadata to attach to traces. There are additional [roles][16] that need to be set in order to allow access to this metadata.
 
-    In the `otel-agent-conf` ConfigMap's `data.otel-agent-config` `service.pipelines.traces` section:
+The [example visible in the repository][11] is complete, ready-to-use and has the correct roles set up. All that is required of you is to provide your [application container][17].
 
-    ```yaml
-      # ...
-      # resourcedetection must come before k8sattributes
-      processors: [batch, resourcedetection, k8sattributes]
-      # ...
-    ```
+To learn more about configuring your application, please see the [Application Configuration](#application-configuration) section below.
 
-3. For OpenTelemetry Collectors in standalone collector mode, which receive traces from downstream collectors and export to Datadog's backend, include a `batch` processor and a `k8sattributes` processor. These should be included along with the `datadog` exporter and added to the `traces` pipeline.
+### Gateway Collector Service
 
-    In the `otel-collector-conf` ConfigMap's `data.otel-collector-config` `processors` section:
+For Gateway deployments, we'll first need to set up each [OpenTelemetry Collector agent][18], just like in the [DaemonSet deployment](#daemonset-deployment). Then, the DaemonSet needs to be changed to include an [OTLP exporter][19] instead of the Datadog Exporter [currently in place][20]:
 
-    ```yaml
-      # ...
-      batch:
-      k8sattributes:
-      # ...
-    ```
+```yaml
+# ...
+exporters:
+  otlp:
+    endpoint: "<GATEWAY_HOSTNAME>:4317"
+# ...
+```
 
-    In the `otel-collector-conf` ConfigMap's `data.otel-collector-config` `exporters` section:
+Then, make sure that the service pipelines use this exporter, instead of the Datadog one that [is in place in the example][21]:
+
+```yaml
+# ...
+    service:
+      pipelines:
+        metrics:
+          receivers: [otlp]
+          processors: [resourcedetection, k8sattributes, batch]
+          exporters: [otlp]
+        traces:
+          receivers: [otlp]
+          processors: [resourcedetection, k8sattributes, batch]
+          exporters: [otlp]
+# ...
+```
 
-    ```yaml
-      exporters:
-        datadog:
-          api:
-            key: <YOUR_API_KEY>
-    ```
+This will ensure that each agent forwards its data via the OTLP protocol to the Collector Gateway. Make sure to replace `GATEWAY_HOSTNAME` with the address of your OpenTelemetry Colllector Gateway.
 
-    In the `otel-collector-conf` ConfigMap's `data.otel-collector-config` `service.pipelines.traces` section:
+Additionally, to ensure that Kubernetes metadata continues to be applied to traces, we need to tell the [`k8sattributes` processor][22] to forward the Pod IP to the Gateway Collector so that it can obtain the metadata:
```

Comment:
```suggestion
5. To ensure that Kubernetes metadata continues to be applied to traces, tell the [`k8sattributes` processor][22] to forward the Pod IP to the Gateway Collector so that it can obtain the metadata:
```